qulice-checkstyle/src/main/java/com/qulice/checkstyle/CheckstyleValidator.java

```diff
@@ -68,7 +69,15 @@ public CheckstyleValidator(final Environment env) {
     public Collection<Violation> validate(final Collection<File> files) {
         final List<File> sources = this.getNonExcludedFiles(files);
         try {
+            Logger.debug(this, "Checkstyle processing files: %s", sources);
```

Comment:
@volodya-lombrozo you really want to print hundreds of file names to the console in one log message?

Reply:
Used `Logger.debug(this, "Checkstyle processing %d files", sources.size());`